load_site.py

```diff
@@ -0,0 +1,27 @@
+import asyncio
+import aiohttp
+import time
+
+
+async def load_yandex():
+    with aiohttp.ClientSession() as s:
+        resp = await s.get('https://ya.ru')
+        print(await resp.json())
+
+
+async def load_google():
+    with aiohttp.ClientSession() as s:
+        resp = await s.get('https://google.com')
+        print(await resp.json())
+
+
+async def main():
+    while True:
```

Comment:
If we want this program to run for as long as possible no matter what, we should wrap its body (inside while True loop) in a try/except block.